src/routes/OverviewPage/DescriptionsCard.tsx

```diff
@@ -46,18 +46,29 @@ function DescriptionsCard() {
           </Icon>
         }
       >
-        <p className="pf-u-mb-lg">
-          Scans images for known vulnerabilities based on specific languages,
-          packages, and image layers. Provides a dashboard highlighting the
-          riskiest image vulnerabilities and deployments.
-        </p>
-        <p>
-          Verifies image signatures against preconfigured keys for image
-          attestation and integrity. Correlates vulnerabilities to running
-          deployments, not just images Enforces policies based on vulnerability
-          details at build time using continuous integration/continuous delivery
-          (CI/CD) integrations.
-        </p>
+        <List>
+          <ListItem>
+            Detects host-level vulnerabilities and potential security threats in
+            Red Hat Enterprise Linux® CoreOS.
+          </ListItem>
+          <ListItem>
+            Scans images for known vulnerabilities based on specific languages,
+            packages, and image layers.
+          </ListItem>
+          <ListItem>
+            Provides a dashboard highlighting the riskiest image vulnerabilities
+            and deployments.
+          </ListItem>
+          <ListItem>
+            Verifies image signatures against preconfigured keys for image
+            attestation and integrity.
+          </ListItem>
+          <ListItem>
+            Correlates vulnerabilities to running deployments, not just images
+            Enforces policies based on vulnerability details at build time using
+            continuous integration/continuous delivery (CI/CD) integrations.
```

Comment:
Looks good 👍 this seems a bit fragmented though? Might be missing punctuation between "images" and "Enforces"